### PR TITLE
Switch to protocol V6 with 0.17

### PR DESCRIPTION
**Pull request by nvie**
Following up on https://github.com/liveblocks/liveblocks-cloudflare/pull/80#pullrequestreview-988717340, but on the client side.

### CODE REVIEW

File: packages/liveblocks-client/src/client.ts [hunk 1]
```diff
@@ -90,7 +90,7 @@ export function createClient(options: ClientOptions): Client {
         WebSocketPolyfill: clientOptions.WebSocketPolyfill,
         fetchPolyfill: clientOptions.fetchPolyfill,
         liveblocksServer:
-          (clientOptions as any).liveblocksServer || "wss://liveblocks.net/v5",
+          (clientOptions as any).liveblocksServer || "wss://liveblocks.net/v6",
```
Comment (nvie) [inline packages/liveblocks-client/src/client.ts:93]: @GuillaumeSalles @ofoucherot Is this all there is to it?

**Review by GuillaumeSalles — APPROVED** [review]

File: packages/liveblocks-client/src/client.ts (hunk 1 above)
Comment (GuillaumeSalles) [reply packages/liveblocks-client/src/client.ts:93]: Yes :)

**Review by ofoucherot — APPROVED** [review]

**Comment by nvie** [comment]
Anyone knows what's up with the super slow e2e tests on this PR? Hope the switch to v6 doesn't trigger this massive performance drop? 😬

**Comment by nvie** [comment]
Also added an index page to more easily discover all the available E2E test pages that exist, when playing with the app manually:

Screen Shot 2022-05-31 at 22 30 31@2x

**Comment by nvie** [comment]
All tests are green again! Thx for pinpointing the v6 issue @ofoucherot and for getting that deployed 🙏

**Comment by nvie** [comment]
Also @GuillaumeSalles great thinking to expose this under v6 😉

**Pull request merged**